### PR TITLE
Refactor: changed the `resolve_event_context` method to human-readable

### PR DESCRIPTION
**Pull request by NiKuma0**
Refactor: changed the `resolve_event_context` method of `UserContextMiddleware` class to human-readable

### CODE REVIEW

**Comment by Olegt0rr** [comment]
Current solution is better than suggested.

File: aiogram/dispatcher/middlewares/user_context.py [hunk 1]
```diff
@@ -43,32 +43,21 @@ def resolve_event_context(cls, event: Update) -> Tuple[Optional[Chat], Optional[
...
+            'chosen_inline_result', 'callback_query',
+            'shipping_query', 'pre_checkout_query',
+            'poll_answer', 'my_chat_member',
+            'chat_member', 'chat_join_request',
+        )
+        for parametr in parametrs:
+            parametr = getattr(event, parametr, None)
+            if parametr:
+                break
+        return getattr(parametr, 'chat', None), getattr(parametr, 'from_user', None)
```
Comment (Olegt0rr) [inline aiogram/dispatcher/middlewares/user_context.py:63]: it's wrong for `poll_answer`

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: aiogram/dispatcher/middlewares/user_context.py [hunk 2]
```diff
@@ -43,32 +43,21 @@ def resolve_event_context(cls, event: Update) -> Tuple[Optional[Chat], Optional[
...
-            return event.channel_post.chat, None
-        if event.edited_channel_post:
-            return event.edited_channel_post.chat, None
         if event.inline_query:
-            return None, event.inline_query.from_user
-        if event.chosen_inline_result:
-            return None, event.chosen_inline_result.from_user
-        if event.callback_query:
-            if event.callback_query.message:
+            if event.inlinee_query.message:
```
Comment (gabbhack) [inline aiogram/dispatcher/middlewares/user_context.py:47]: typo

**Review by gabbhack — CHANGES_REQUESTED** [review]

**Comment by NiKuma0** [comment]
> Current solution is better than suggested.

Why? 
Okay, I understand that maybe my code will run slower than before (by quite a bit). But it's really the most human readable.

**Comment by JrooTJunior** [comment]
> Okay, I understand that maybe my code will run slower than before (by quite a bit). But it's really the most human readable.

We have already tested such a solution 10 months ago. This solution is slower than current by 4 times and this is due to the use of a cycle with the selection of redundant parameters instead of direct access.

The processing speed might be critical for high-loaded bots

As The Zen of Python says - Explicit is better than implicit.

About readability, i don't thing the for-loop with `gettattr` is more readability instead of plain if-conditions.

---

Also your code doesn't follows project code-style (black)

**Pull request closed without merging**